api/utils/export.py

```diff
@@ -0,0 +1,26 @@
+from enum import Enum
+
+
+class ExportType(Enum):
```

Comment:
But is it used anywhere? Sorry if I'm blind!

Reply:
Can't see it used in the larger PR either

Reply:
I think this came from a bad merge (when I split the PRs), this was refactored into `extensions.py` which covered more use cases.

Reply:
I've removed the file now, thanks!